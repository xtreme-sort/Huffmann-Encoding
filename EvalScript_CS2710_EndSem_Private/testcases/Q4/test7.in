GenerateLengthLimitedHuffman
zqjanmpjudgpjpyteworshcjikzhociqogsmxlckyvjqxcdxlfemrxezznjmxykadsirccpzgdndhhmeltvrwilxnhgdvhekcnfxmsujbytytvfqygzuvtnmaxskfplpvehlrhblepyhueyylkitxdlpralkmfidmwgzitwsbbvfgptbmsldwjbfsvjozyocqtqaxduknghtdpnzkoqsxdyxurnhlrxhezqrkicojcobspcijhokdpikxfqhseyuosnpqadaetkrymsdhsdqhlqtzjfmqxwxksnvqkeqvjonhbcsemeazkgpamucvcwthmmmeovhruchwmmdbmumujhikxzugacpiwzyblhvesatqyyocs
9
Quit
