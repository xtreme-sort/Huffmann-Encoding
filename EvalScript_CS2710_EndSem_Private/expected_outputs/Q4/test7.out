c 0000
d 0001
h 0010
k 0011
m 0100
s 0101
a 01100
b 01101
e 01110
f 01111
g 10000
i 10001
j 10010
l 10011
n 10100
o 10101
p 10110
q 10111
r 11000
t 11001
u 11010
v 11011
w 11100
x 11101
y 11110
z 11111
cost: 1738
